specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/keyvault.json

```diff
@@ -0,0 +1,4834 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The issuer for the key vault certificate.",
+            "schema": {
+              "$ref": "#/definitions/IssuerBundle"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/KeyVaultError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Certificates"
+        ],
+        "operationId": "DeleteCertificateIssuer",
+        "summary": "Deletes the specified certificate issuer.",
+        "description": "The DeleteCertificateIssuer operation permanently removes the specified certificate issuer from the vault. This operation requires the certificates/manageissuers/deleteissuers permission.",
+        "parameters": [
+          {
+            "name": "issuer-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the issuer."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The issuer for the key vault certificate.",
+            "schema": {
+              "$ref": "#/definitions/IssuerBundle"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/KeyVaultError"
+            }
+          }
+        }
+      }
+    },
+    "/certificates/{certificate-name}/create": {
+      "post": {
+        "tags": [
+          "Certificates"
+        ],
+        "operationId": "CreateCertificate",
+        "summary": "Creates a new certificate.",
+        "description": "If this is the first version, the certificate resource is created. This operation requires the certificates/create permission.",
+        "parameters": [
+          {
+            "name": "certificate-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[0-9a-zA-Z-]+$",
+            "description": "The name of the certificate."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/CertificateCreateParameters"
+            },
+            "description": "The parameters to create a certificate."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Created certificate bundle.",
+            "schema": {
+              "$ref": "#/definitions/CertificateOperation"
+
```

Comment:
This is marked as read-only however the property is also required (like 4758).  It can't be both, which one is correct?

Reply:
I'm not sure what you're referring to here. Is this comment on the wrong line?

Reply:
Was supposed to be on line 4742, not sure what happened.